sqlchain/chain.go

```diff
@@ -1020,6 +1020,10 @@ func (c *Chain) AddResponse(resp *types.SignedResponseHeader) (err error) {
 	return c.ai.addResponse(c.rt.getHeightFromTime(resp.GetRequestTimestamp()), resp)
 }
 
+func (c *Chain) GetPeerLeaderID() proto.NodeID {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Chain.GetPeerLeaderID should have comment or be unexported